.github/workflows/post-pr-cleanup.yml

```diff
@@ -0,0 +1,61 @@
+name: cleanup-codecov-reports
+
+
+# This workflow deletes the html reports for the given PR to tidy up the helper repository
+
+on:
+  pull_request:
+    types: [closed]
+    branches: [main]
+
+
+jobs:
+  cleanup:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          repository: dopeshot/beyond-life-codecov
+          token: ${{ secrets.BOT_PAT }}
+          ref: main
+
+      - name: Extract branch name
+        run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/} | tr '/' '_')"
+        id: extract_branch
+
+      - name: cleanup html reports
+        run: rm -rf  ${{ steps.extract_branch.outputs.branch }} || true
+
+      - name: Commit

```

Comment:
KI schlägt folgendes vor:

The workflow commits the changes to the main branch. This may not be desirable, as it could overwrite changes that have been made to the branch since the pull request was closed. You may want to consider committing the changes to a separate branch, or using a different workflow to push the changes to the main branch.

Reply:
Macht Sinn, aber das externe Repo is ja nur hierfür da, passt also